test_scripts/WebEngine/WebEngineProjection/010_OnSCU_after_Show_widget.lua

```diff
@@ -0,0 +1,83 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0273-webengine-projection-mode.md
+--
+-- Description: Check that SDL transfer `OnSystemCapabilitiesUpdated` notification from HMI to an App
+-- which sent `Show` RPC with `templateConfiguration` parameter for widget window after `Show` response
+--
+-- Preconditions:
+-- 1) SDL and HMI are started
+-- 2) App is registered
+-- 3) App creates new widget
+-- Steps:
+-- 1) App sends `Show` request with `templateConfiguration` for widget window
+-- SDL does:
+--  - proceed with `Show` request successfully
+--  - not send `OnSystemCapabilityUpdated` to App
+-- 2) HMI sends `OnSystemCapabilityUpdated` to SDL
+-- SDL does:
+--  - transfer `OnSystemCapabilityUpdated` notification to App
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/WebEngine/commonWebEngine')
+
+--[[ Local Variables ]]
+local createWindowParams = {
+  windowID = 1,
+  windowName = "Name",
+  type = "WIDGET"
+}
+
+local appSessionId = 1
+local appHMIType = "WEB_VIEW"
+
+--[[ General configuration parameters ]]
+config.application1.registerAppInterfaceParams.appHMIType = { appHMIType }
+config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 7
+config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 0
+config.defaultMobileAdapterType = "WS"
+common.testSettings.restrictions.sdlBuildOptions = {{ webSocketServerSupport = { "ON" }}}
+
+--[[ Local Functions ]]
+local function sendShow()
+  local showParams = {
+    mainField1 = "MainField1",
+    windowID = createWindowParams.windowID,
+    templateConfiguration = {
+      template = "Template1"
+    }
+  }
```

Comment:
Do we need to create this object each time on `sendShow` call? I think this structure should be placed near `createWindowParams`.